lib/rubocop/cop/rspec/describe_class.rb

```diff
@@ -44,12 +44,20 @@ class DescribeClass < Cop
 
         def_node_matcher :shared_group?, SharedGroups::ALL.block_pattern
 
-        def on_top_level_describe(node, args)
+        def on_top_level_describe(node, (described_value, _))
           return if shared_group?(root_node)
           return if valid_describe?(node)
           return if describe_with_rails_metadata?(node)
+          return if string_constant_describe?(described_value)
 
-          add_offense(args.first)
+          add_offense(described_value)
+        end
+
+        private
+
+        def string_constant_describe?(described_value)
+          described_value.str_type? &&
+            described_value.value =~ /^((::)?[A-Z]\w*)+$/
```

Comment:
Could you please update the cop documentation with another “good” example, e.g. `'Some::Thing'`?

Reply:
Thank you, I forgot about this, added.

Also with `subject { Object.const_get(self.class.description) }` — it can be helpful for others.

And I've added PR template checkbox about documentation in the same commit — I hope it's OK.